iphone/TitaniumKit/TitaniumKit/Sources/API/TiBlob.m

```diff
@@ -111,6 +111,26 @@ - (NSUInteger)size
 }
 GETTER_IMPL(NSUInteger, size, Size);
 
+- (id)_initWithPageContext:(id<TiEvaluator>)pageContext
+{
+  return [[TiBlob alloc] init];
```

Comment:
The default `_initWithPageContext: `constructor should be re-added to the parent class (`TiViewProxy`) again, since other proxies (especially in modules) may be constructed that way as well.

Reply:
ok, I can do that too, I suppose.

Related: why do we need to alloc calls here? Typical usage before was:
```obj-c
[[TiBlob alloc] _initWithPageContext:context];
```

so should the redirecting version just be:
```obj-c
- (id)_initWithPageContext:(id<TiEvaluator>)pageContext
{
  return [self init];
}
```

Reply:
You're right, no alloc required!